src/sqlancer/feldera/ast/FelderaAggregate.java

```diff
@@ -13,30 +13,32 @@ public class FelderaAggregate implements FelderaExpression {
     private boolean blackbox;
 
     public enum FelderaAggregateFunction {
-        AVG(FelderaSchema.FelderaDataType.INT, FelderaSchema.FelderaDataType.DOUBLE),
+        AVG(FelderaSchema.FelderaDataType.INT, FelderaSchema.FelderaDataType.FLOAT),
```

Comment:
we don't even support FLOAT in SQL, this is very confusing.

Reply:
With FLOAT, we represent both DOUBLE and REAL here.